src/pocketdb/repositories/CheckpointRepository.cpp

```diff
@@ -361,6 +361,1138 @@ namespace PocketDb
         _socialCheckpoints.emplace_back("ece0d09e1e89e15582f85e90a1d4df2e5efa9d96e3a2d15dcc9648b4db814ac7", 300, 4);
         _socialCheckpoints.emplace_back("f9e1d693b97da9688b9e1353a334aa167d5134a087e690bd462bb9157e000f69", 300, 4);
         _socialCheckpoints.emplace_back("6fcad0f123d4781dd846076a705aadd9c4f347920a76d88dfbe9905777afb8ae", 303, 8);
+        _socialCheckpoints.emplace_back("019d34809560f74e7ba065b1943eb241d29ba5427caefdf922774610c07521af", 100, 6);
```

Comment:
Please describe what is this

Reply:
For users, we changed the rules of name processing or the order of editing. The old rules were eliminated to make the code easier and checkpoints were added to ensure synchronization. Validation of the data is maintained at the OP_RETURN validation level